cover

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+#
+# Generate coverage HTML for a package
+# e.g. PKG=./unit ./cover
```

Comment:
Ugg, I know this comes from our standard goo scripts but I find it completely irritating to the point that I never use it.

 - PKG should be provided as an argument, not environment.
 - The default behavior should be to scan all packages, if the go coverage tool is really incapable of reporting multiple packages at once then loop over the packages running go test once for each.